__tests__/Unit/Components/Tabs/Tab.test.tsx

```diff
@@ -1,13 +1,18 @@
 import Tabs from '@/components/Tabs';
 import { fireEvent, render, screen } from '@testing-library/react';
-import { Tab, TABS } from '@/interfaces/task.type';
-import { COMPLETED, DONE, AVAILABLE, UNASSINGED } from '@/constants/constants';
+import {
+    Tab,
+    TABS,
```

Comment:
`TAB` and `Tab`, could we change them with a descriptive name to get the differences?

Reply:
We may have to raise another PR for these changes, as we are using these in multiple files and those changes could be out of scope for this PR.